kornia/geometry/transform/elastic_transform.py

```diff
@@ -17,6 +18,7 @@ def elastic_transform2d(
     alpha: Tuple[float, float] = (1.0, 1.0),
     align_corners: bool = False,
     mode: str = 'bilinear',
+    padding_mode: Union[str, int, SamplePadding] = SamplePadding.ZEROS.name
```

Comment:
```suggestion
    padding_mode: str = 'zeros'
```
torchscript doesn't support well yet `Union` type

Reply:
just change to str for now?

Reply:
Yeah